src/julia.h

```diff
@@ -741,10 +741,11 @@ typedef struct _jl_binding_t {
     _Atomic(jl_binding_partition_t*) partitions;
     jl_array_t *backedges;
     uint8_t did_print_backdate_admonition:1;
+    uint8_t did_print_implicit_import_admonition:1;
```

Comment:
https://github.com/JuliaLang/julia/blob/master/base/invalidation.jl#L116 currently assumes the bitposition of the export flag which this changes. I'm about to refactor that code to no longer use the bitset (which is implementation defined anyway), but if you want to merge this change before then, you'll have to adjust one or the other.

Reply:
Thanks for the pointer. I will probably just fix the offset for now. Immediately after this PR, I want to change this to an atomics flags field so we can use atomic-and / atomic-or to set these flags safely.